source/blog/2016-01-07-atomic-diff.html.md

```diff
@@ -0,0 +1,136 @@
+---
+title: atomic diff
+author: baude
+date: 2016-01-07 20:34:42 UTC
+published: false
+comments: true
+---
+
+The original mission of the [atomic application](https://github.com/projectatomic/atomic) to install, manage, and run container images using container Labels, has slowly been growing.  We have been concentrating on ease of use and value-add functions for containers and images. We recently added the `atomic diff` command.  This command allows you to differentiate between two container images. This is critically important if you are like me and you have hundreds of container images on your system. I also use it when I pull an updated image from a repository and want to see what has changed.
+
+**Note:** While docker itself has a subcommand called diff, it is only useful when trying to determine the difference between a container and its image.  It does not compare two different docker objects looking for differences amongst them.
+
```

Comment:
From this point on, you no longer talk about images, but "objects". I'm guessing a docker object can be either a container or an image? It might help to clarify this.

Maybe in that same **Note:** ? E.g.

**Note:** While docker itself has a subcommand called diff, it is only useful when trying to determine the difference between a container and its image.  On the other hand, the atomic diff command can compare across any two docker objects (images or containers).